packages/alias/src/index.ts

```diff
@@ -25,9 +25,7 @@ function matches(pattern: string | RegExp, importee: string) {
   return importeeStartsWithKey && importeeHasSlashAfterKey;
 }
 
-function normalizeId(id: string): string;
-function normalizeId(id: string | undefined): string | undefined;
-function normalizeId(id: string | undefined) {
+function normalizeId<T>(id: T) {
```

Comment:
This should be restricted to just strings
```suggestion
function normalizeId<T extends string | undefined>(id: T) {
```

Reply:
makes sense. done